yt_dlp/extractor/twitter.py

```diff
@@ -514,6 +567,102 @@ class TwitterIE(TwitterBaseIE):
         },
         'playlist_count': 4,
         'params': {'skip_download': True},
+    }, {
+        'url': 'https://twitter.com/MesoMax919/status/1575560063510810624',
+        'info_dict': {
+            'id': '1575559336759263233',
+            'display_id': '1575560063510810624',
+            'ext': 'mp4',
+            'title': 'Max Olson - Absolutely heartbreaking footage captured by our surge probe of catastrophic storm surge washing away homes. I have never seen anything like this. We have now left the area as hoards of emergency crew have arrived. #HurricaneIan   FULL VIDEO -',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'description': 'Absolutely heartbreaking footage captured by our surge probe of catastrophic storm surge washing away homes. I have never seen anything like this. We have now left the area as hoards of emergency crew have arrived. #HurricaneIan   FULL VIDEO - https://t.co/DOJJn2VThV https://t.co/iPBUyVKw4s',
```

Comment:
use md5 for long fields

Reply:
For this field yes, though for 18 and 19 I think it makes sense to leave the string so you can compare and see that only the link is missing.